packages/shared-base/src/Persona/type.ts

```diff
@@ -11,11 +11,14 @@ export interface PersonaInformation {
     linkedProfiles: ProfileInformation[]
 }
 export interface ProfileInformation {
+    [index: string]: unknown
     /** The nickname of the profile. Should be used in prior. */
     nickname?: string
     avatar?: string
     identifier: ProfileIdentifier
     fingerprint?: string
+    address?: string
+    fromNextID?: boolean
```

Comment:
Can you explain this field?

Reply:
to divide local data and NextID data in the SelectRecipietsDialog